### DISCUSSION TITLE
Why canvas tag not adopt tailwind css ?

### DISCUSSION THREAD

**Discussion by magic-thomas** [tailwindlabs/tailwindcss#3192] [Help] [answered]
Hi . I tried tailwindcss in canvas tag. 

```html

<canvas id="myChart" class="bg-red-300 w-96 h-48" ></canvas>

```

but it is not working .

it is intended? . should I use pure css for canvas tag ?

Is there way to code in base layer?

**Comment by MathGeniusJodie** [comment, marked as answer]
works for me
https://play.tailwindcss.com/tuOzmr6z6E
